packages/cdk/lib/constructs/launch-template-data.ts

```diff
@@ -0,0 +1,112 @@
+import { APP_NAME } from "../constants";
+
+export class LaunchTemplateData {
+  /**
+   * WARNING! Changing the content of the launch template user data WILL NOT cause the redeployment of any AWS Batch
+   * compute environments that use the template. Ensure you create a new Compute Environment by, for example, running:
+   * agc context destroy --all
+   * agc context deploy <context-name>
+   */
+  static renderLaunchTemplateData(workflowOrchestrator: string): string {
+    return `MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="==MYBOUNDARY=="
+
+--==MYBOUNDARY==
+Content-Type: text/cloud-config; charset="us-ascii"
+
+packages:
+- jq
+- grep
+- btrfs-progs
+- sed
+- git
+- unzip
+- amazon-cloudwatch-agent
+
+write_files:
+- permissions: '0644'
+  path: /opt/aws/amazon-cloudwatch-agent/etc/config.json
+  content: |
+    {
+      "agent": {
+        "logfile": "/opt/aws/amazon-cloudwatch-agent/logs/amazon-cloudwatch-agent.log"
+      },
+      "logs": {
+        "logs_collected": {
+          "files": {
+            "collect_list": [
+              {
+                "file_path": "/opt/aws/amazon-cloudwatch-agent/logs/amazon-cloudwatch-agent.log",
+                "log_group_name": "/aws/ecs/container-instance/${APP_NAME}",
+                "log_stream_name": "/aws/ecs/container-instance/${APP_NAME}/{instance_id}/amazon-cloudwatch-agent.log"
+              },
+              {
+                "file_path": "/var/log/cloud-init.log",
+                "log_group_name": "/aws/ecs/container-instance/${APP_NAME}",
+                "log_stream_name": "/aws/ecs/container-instance/${APP_NAME}/{instance_id}/cloud-init.log"
+              },
+              {
+                "file_path": "/var/log/cloud-init-output.log",
+                "log_group_name": "/aws/ecs/container-instance/${APP_NAME}",
+                "log_stream_name": "/aws/ecs/container-instance/${APP_NAME}/{instance_id}/cloud-init-output.log"
+              },
+              {
+                "file_path": "/var/log/ecs/ecs-init.log",
+                "log_group_name": "/aws/ecs/container-instance/${APP_NAME}",
+                "log_stream_name": "/aws/ecs/container-instance/${APP_NAME}/{instance_id}/ecs-init.log"
+              },
+              {
+                "file_path": "/var/log/ecs/ecs-agent.log",
+                "log_group_name": "/aws/ecs/container-instance/${APP_NAME}",
+                "log_stream_name": "/aws/ecs/container-instance/${APP_NAME}/{instance_id}/ecs-agent.log"
+              },
+              {
+                "file_path": "/var/log/ecs/ecs-volume-plugin.log",
+                "log_group_name": "/aws/ecs/container-instance/${APP_NAME}",
+                "log_stream_name": "/aws/ecs/container-instance/${APP_NAME}/{instance_id}/ecs-volume-plugin.log"
+              }
+            ]
+          }
+        }
+      }
+    }
+
+runcmd:
+
+# start the amazon-cloudwatch-agent
```

Comment:
Can you please add the change for https://github.com/aws/amazon-genomics-cli/pull/301 here as well?

Reply:
Done